src/keeper.ts

```diff
@@ -468,17 +469,22 @@ class Keeper {
     // sort unknown liq prices by leverage
     unknownLiqPrice.sort((p1, p2) => p2.leverage - p1.leverage); //desc
 
+    const outdatedLiqPrices = liqPriceFar.filter(
+      p =>
+        p.liqPriceUpdatedTimestamp <
+        this.blockTipTimestamp - farPriceRecencyCutoff
+    );
     // sort far liquidation prices by how out of date they are
     // this should constantly update old positions' liq price
-    liqPriceFar.sort(
+    outdatedLiqPrices.sort(
       (p1, p2) => p1.liqPriceUpdatedTimestamp - p2.liqPriceUpdatedTimestamp
     ); //asc
 
     // first known close prices, then unknown prices yet
     return [
       liqPriceClose, // all close prices within threshold
       unknownLiqPrice, // all unknown liq prices (to get them updated)
-      liqPriceFar.slice(0, maxFarPricesToUpdate), // some max number of of outdated prices to reduce spamming the node and prevent self DOS when there are many positions
+      outdatedLiqPrices.slice(0, maxFarPricesToUpdate), // some max number of of outdated prices to reduce spamming the node and prevent self DOS when there are many positions
```

Comment:
Only updating 1 of the current positions on purporse?

Reply:
So with the way we're calling liquidationGroups currently this PR doesn't change anything? Or I guess there might be 0 positions after the filter
Hmm wouldn't it just be one postion that gets updated? 
```
 liquidationGroups(
    posArr: Position[],
    priceProximityThreshold = 0.05,
    maxFarPricesToUpdate = 1, // max number of older liquidation prices to update
    farPriceRecencyCutoff = 6 * 3600 // interval during which the liquidation price is considered up to date if it's far
  )
```
`this.liquidationGroups(openPositions);`

Reply:
yes, after this update, the last group might be empty if no positions need updating